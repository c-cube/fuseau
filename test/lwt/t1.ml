let pf = Printf.printf
let ( let@ ) = ( @@ )

let run () =
  Fuseau_lwt.main @@ fun () ->
  pf "start\n%!";
  Fuseau.sleep_s 0.5;
  pf "done\n%!";
  42

let () =
  let@ () = Trace_tef.with_setup () in
  let n = run () in
  pf "res: %d\n" n
