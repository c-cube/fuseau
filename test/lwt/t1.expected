start
done
res: 42
